<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendForm</class>
 <widget class="QDialog" name="sendForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>193</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>193</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>193</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Перевод средств</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/logo.ico</activeon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="infoLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Внимание! При переводе средств взымается комиссия 1.5%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="numLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Номер счета:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="dateLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Срок действия: MM/ГГ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="numEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="McomboBox">
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="YcomboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sumLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Сумма BYN:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="sumEdit"/>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Перевести</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
